source/_docs/scripts.markdown

```diff
@@ -5,7 +5,7 @@ toc: true
 no_toc: true
 ---
 
-Scripts are a sequence of actions that Home Assistant will execute. Scripts are available as an entity through the standalone [Script component] but can also be embedded in [automations] and [Alexa/Amazon Echo] configurations.
+Scripts are a sequence of {% term actions %} that Home Assistant will execute. Scripts are available as an entity through the standalone [Script component] but can also be embedded in {% term automations %} and [Alexa/Amazon Echo] configurations.
```

Comment:
Should we add something to <https://developers.home-assistant.io/docs/documenting> about when to use this?

Reply:
Going to open a PR for that.

Reply:
Done: https://github.com/home-assistant/developers.home-assistant/pull/1710

Reply:
Thanks 👍